ui/mirage/factories/job.js

```diff
@@ -269,36 +285,6 @@ export default Factory.extend({
         });
     }
 
-    if (!job.shallow) {
-      const knownEvaluationProperties = {
-        job,
-        namespace: job.namespace,
-      };
-      server.createList(
-        'evaluation',
-        faker.random.number({ min: 1, max: 5 }),
-        knownEvaluationProperties
-      );
-      if (!job.noFailedPlacements) {
-        server.createList(
-          'evaluation',
-          faker.random.number(3),
-          'withPlacementFailures',
-          knownEvaluationProperties
-        );
-      }
-
-      if (job.failedPlacements) {
-        server.create(
-          'evaluation',
-          'withPlacementFailures',
-          assign(knownEvaluationProperties, {
-            modifyIndex: 4000,
-          })
-        );
-      }
-    }
```

Comment:
This might be breaking some tests. Try running `yarn local:exam` this is the fastest way to run our test suite.